bin/oneoff/data_fix/reset_race_data

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+#
+# Script to reset the race data for the subset of users affected by the race
+# interstitial bug
+#
+
+require_relative '../../../dashboard/config/environment'
+
+File.readlines(ARGV[0]).each do |line|
```

Comment:
It's probably worth rescuing any error that is raised within this block.  Log the id that failed and then move on to the next one.